src/components/Panel.tsx

```diff
@@ -1,45 +1,75 @@
 import * as React from "react";
 import { GenericWedgeIcon } from "@nypl/dgx-svg-icons";
 
+// The Panel component comes with five built-in styles: "success" (green),
+// "warning" (yellow), "danger" (red), "instruction" (blue), and "default" (gray).
+// You can also define your own styles.  If you do not pass in a value for the style prop,
+// the style will be set to "default".
+
+// There are two ways to populate the panel with content: you can use either the body prop,
+// which accepts a JSX element, or the text prop, which accepts an HTML string.
```

Comment:
Is at least one of them required? I think you should include that, and as I wrote below, make that programmatically true.

Reply:
I'm not sure whether they should be required.  On the one hand, I feel like we should build as much flexibility as possible into these components...but on the other hand, I am having a lot of trouble coming up with a realistic scenario in which someone would actually want to render an empty panel.  What do you think?

Reply:
I don't think anyone would want an empty panel. I'm talking more of making either or programmatically required so that it's not accidentally empty in case no props are passed in. It would also be helpful so you don't pass in both props and expect to see them.